giskard/ml_worker/websocket/listener.py

```diff
@@ -563,7 +563,7 @@ def run_test_suite(
                 test_args: Dict[str, Any] = copy(test_args)
                 test_args.update(**test_args.pop("kwargs"))
             updated_test_args.append(test_args)
-            suite.add_test(t["test"].get_builder()(**test_args), t["id"])
```

Comment:
Why aren't we using builder now?

Reply:
The `__call__` method now always makes a copy of the object using `instance = copy.deepcopy(self)` so we don't need the `get_builder` method anymore.